concepts/case/about.md

```diff
@@ -0,0 +1,129 @@
+# Case
+
+[Case][case] (often referred to as switch in other languages) is a form of control expression like if-else.
+Case allows for chaining of multiple if-else-if statements and can be more readable and also allows for powerful constructs.
```

Comment:
```suggestion
Case allows for chaining of multiple `when` statements and can be more readable and also allows for powerful constructs.
```

Also, does it really allow for "more powerful constructs"?  Or are they just as powerful but different?

Reply:
It doesnt really say they are more powerful just that they allow for powerful constructs.
And it is suppose to be if-else-if.

Reply:
I can see what it says, if "it" is this copy.  But should it?

Reply:
And, yes, while we can indeed use if-else-end along and even inside of case statements, the case statement does not itself use that syntax.

Reply:
I would even say that it is allowed for, even without case.  So the case statement isn't what is allowing for chaining if-else-if statements.